lib/api/model/model.dart

```diff
@@ -266,7 +276,8 @@ class ZulipStream {
     required this.historyPublicToSubscribers,
     required this.messageRetentionDays,
     required this.streamPostPolicy,
-    required this.canRemoveSubscribersGroupId,
+    required this.canRemoveSubscribersGroup,
+    required this.streamWeeklyTraffic,
```

Comment:
nit:

    api: Update ZulipStream for api changes

"for API changes" — in English as opposed to code, "API" is an initialism written in all caps.